src/test/java/org/opensearch/security/transport/SecurityInterceptorTests.java

```diff
@@ -128,7 +132,8 @@ public void setup() {
             clusterService,
             sslExceptionHandler,
             clusterInfoHolder,
-            sslConfig
+            sslConfig,
+            this::testActionTraceSupplier
```

Comment:
Super tiny nit, you can inline this function if you aren't testing the alternative case
```suggestion
            () -> true
```